pulpcore/tasking/tasks.py

```diff
@@ -178,15 +178,15 @@ def enqueue_with_reservation(func, resources, args=None, kwargs=None, options=No
         options = dict()
 
     resources = {util.get_url(resource) for resource in resources}
-    inner_task_id = str(uuid.uuid4())
+    inner_job_id = str(uuid.uuid4())
     redis_conn = connection.get_redis_connection()
     current_job = get_current_job(connection=redis_conn)
     parent_kwarg = {}
     if current_job:
-        current_task = Task.objects.get(id=current_job.id)
+        current_task = Task.objects.get(job_id=current_job.id)
```

Comment:
Same issue with 'id'

Reply:
This is the `rq` one from `get_current_job` so this one is not in our control.